Utils/Dataflow/pyDKB/dataflow/stage/ProcessorStage.py

```diff
@@ -280,6 +291,24 @@ def process(stage, input_message):
         """
         raise NotImplementedError("Stage method process() is not implemented")
 
+    @staticmethod
+    def skip_process(stage, input_message):
+        """ Mark input message as incomplete and output.
+
+        May be re-implemented just like `process()`,
+        yet has default implememtation.
```

Comment:
ImplemeNtation.

Reply:
This change belongs to #260, which was already approved and just merged. :(
Will fix in a separate PR.

Reply:
Update: done (#266)